Automattic/sensei
# Fix aligning issues for course list filters, align left by default

Fixes https://github.com/Automattic/sensei/issues/5659

### Changes proposed in this Pull Request

* Aligned course filters to left by default, fixed UI breaking issue for post template when aligned left or right.

### Testing instructions
- Add a course list block and select a pattern
- Make sure the filter block is there and filters are aligned to left
- Make sure the design stays unbroken when filters are aligned to both left and right

### Screenshot / Video

![image](https://user-images.githubusercontent.com/6820724/189892756-71efc0a3-9421-4cba-ab25-c4f17fa0cbb1.png)
